docs/_embeds/apis/rust/consumer-auto-offsets.rs

```diff
@@ -0,0 +1,34 @@
+use async_std::stream::StreamExt;
+
+use fluvio::{
+    consumer::{ConsumerConfigExtBuilder, OffsetManagementStrategy},
+    Fluvio, Offset,
+};
+  
+const TOPIC_NAME: &str = "hello-rust";
+const PARTITION_NUM: u32 = 0;
+const CONSUMER_OFFSET: &str = "consumer-auto";
+
+#[async_std::main]
+async fn main() {
```

Comment:
this can be `async fn main() -> Result<()>`

Reply:
Hmm, I don't want to do error mapping.